<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task</class>
 <widget class="QWidget" name="task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;任务管理&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="choiceBox">
      <item>
       <property name="text">
        <string>姓名</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>任务名</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameLi">
      <property name="placeholderText">
       <string>请输入名称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>发布任务</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="taskList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>631</width>
     <height>261</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>任务编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>任务名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>任务周期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>任务状态</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>负责人</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
